### PR TITLE
rtl notification

### PR DESCRIPTION
**Pull request by Erin007**
Notifications, such as the announcement on the teacher homepage, previously didn't switch RTL for RTL languages; now, they do. 

Before: 
notification-rtl-before

After: 
notification-rtl-after

### CODE REVIEW

File: apps/src/templates/studioHomepages/JoinSectionNotifications.jsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ const JoinSectionSuccessNotification = ({sectionName}) => (
     notice={i18n.sectionsNotificationSuccess()}
     details={i18n.sectionsNotificationJoinSuccess({sectionName})}
     dismissible={true}
+    isRtl={false}
```
Comment (breville) [inline apps/src/templates/studioHomepages/JoinSectionNotifications.jsx:32]: Should these maybe use the real isRtl value?

Comment (Erin007) [reply apps/src/templates/studioHomepages/JoinSectionNotifications.jsx:32]: Yes, but they don't have access to that prop yet. I was going to hook it up to the store with the Redux work and switch to the real value then.

Comment (breville) [reply apps/src/templates/studioHomepages/JoinSectionNotifications.jsx:32]: Okay, that sounds fine.

**Comment by Erin007** [comment]
cc: @tanyaparker

**Pull request merged**